src/Servers/IIS/IIS/test/IIS.Shared.FunctionalTests/ApplicationInitializationTests.cs

```diff
@@ -56,6 +56,7 @@ public async Task ApplicationPreloadStartsApp(HostingModel hostingModel)
     [RequiresNewHandler]
     [InlineData(HostingModel.InProcess)]
     [InlineData(HostingModel.OutOfProcess)]
+    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/42658")]
```

Comment:
Again, why aren't we using retry?
https://github.com/dotnet/aspnetcore/pull/42659#issuecomment-1183398649

Reply:
It is already using retry. Every IIS test implicitly uses it: https://github.com/dotnet/aspnetcore/blob/aceefcdf7061d87ff91e4ebd765ddee9b3d9b1a4/eng/test-configuration.json#L12

Reply:
Looks like the failure message is too specific https://github.com/dotnet/aspnetcore/blob/aceefcdf7061d87ff91e4ebd765ddee9b3d9b1a4/eng/test-configuration.json#L13

Reply:
How does the failure message rule come into play?

Aren't these rules all "or". e.g. retry on testName equals X, or testName equals Y, or assemblyName equals Z. The assembly name contains "IIS" so it is retried.

I can see the test being retried multiple times here: https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-aspnetcore-refs-pull-42731-merge-56a44a6721934e368f/IIS.FunctionalTests--net7.0/4/console.5055be8d.log?helixlogtype=result

Reply:
> Aren't these rules all "or".

Yes, the inner rules are `||`ed together. There's a syntax for `&&` but we don't use it. (dotnet/arcade#9597 includes details if you want them).

I did some querying because this has impacted a couple of PRs I've seen and the problem was visible back in April for a few days but went away completely. It restarted on the 8th for some reason. Need someone like @Tratcher or @HaoK to look at why.

If you like Kusto:
``` kusto
let start = 90d;                   // Timespan value
let end = 0m;                      // Timespan value
let repo = "dotnet/aspnetcore";    // Optional: set to empty string if you want results from all repositories
let testName="";                   // Optional: set to empty string if you want results about all tests in the repo
let message="Wrong physical path"; // Controls use of has_all()
AzureDevOpsTests
| where iff(repo == "", Repository == Repository, Repository == repo)
    and iff(testName == "", TestName == TestName, TestName == testName)
    and iff(message == "", Message == Message, Message has_all("Wrong", "physical", "path"))
    and Branch == 'main'
    and Outcome == "Failed" and RunCompleted >= ago(start) and RunCompleted <= ago(end)
| project Repository, BuildId, RunCompleted, TestName, Message
| order by RunCompleted asc 
```

Reply:
Put another way, please go ahead w/ quarantining and we can all ignore the problem until @Tratcher or @HaoK plug away at #42658.